Makefile

```diff
@@ -33,31 +45,37 @@ build: assets build-linux build-windows build-osx
 install:
 ifeq ($(BUILDER_GOOS_GOARCH),"darwin_amd64")
 	@$(MAKE) build-osx
+	@$(MAKE) install-gopath
```

Comment:
I think instead of the build and move step, we should just use "go install" here. It's little bit of more code, but it directly maps to the go commands. So we wouldn't need to move the binaries manually and we can get rid of mentioning GOPATH here. Because the Go toolchain can select from using GOBIN or GOPATH/bin according to what the user has set, and the Makefile would respect that.

Reply:
I see, sounds reasonable.